examples/datacenters/local.arc

```diff
@@ -0,0 +1,125 @@
+variable "account" {
```

Comment:
Any reason we don't want to use the `.hcl` file extension?

Reply:
Most things that are `hcl` do not use `.hcl` they use their own like with terraforms `.tf`. In the long term this makes syntax tools work better because `.arc` would be a unique file extension.